bigquery/samples/tests/conftest.py

```diff
@@ -25,6 +25,12 @@ def client():
     return bigquery.Client()
 
 
+@pytest.fixture
+def one_day_ms(client):
```

Comment:
This doesn't need to be a fixture. Just a normal variable/constant is fine. Fixtures are more for setup & clean-up code.

Reply:
Actually, now that I see where this is used in the sample code, I'd prefer this code stay in the sample. That way it's clear to someone reading the sample what expected values for expiration look like.

Reply:
Took this out of the fixture and put it back into the sample and test.